docs/data_usage_guides/affected_people.md

```diff
@@ -2,29 +2,114 @@
 
 ---
 
+## Internally-Displaced Persons <a id=”idps”></a>
+
+The data for this sub-category is taken from the
+[International Organization for Migration](https://www.iom.int/) (IOM)'s
+[displacement tracking matrix](https://dtm.iom.int/) (DTM),
+which collects and analyzes data on population movements,
+displacements, and mobility patterns to provide timely information
+for humanitarian response efforts.
+THe DTM data used in HAPI is taken from their
+[publicly accessible API](https://dtm.iom.int/data-and-analysis/dtm-api),
+however we note that there are many country-specific DTM datasets
+[available on HDX](https://data.humdata.org/dataset/?dataseries_name=IOM%20-%20DTM%20Baseline%20Assessment&dataseries_name=IOM%20-%20DTM%20Event%20and%20Flow%20Tracking&dataseries_name=IOM%20-%20DTM%20Site%20and%20Location%20Assessment&organization=international-organization-for-migration&q=&sort=last_modified%20desc&ext_page_size=25),
+which contain more detail and disaggregation.
+
+### Summary
+
+{{ read_yaml('data_usage_guides/subcategory_details/idps_details.yaml') }}
+
+### Parameters Returned
+
+The table below describes the parameters returned from this endpoint.
+For available query parameters, please see the
+[API sandbox](https://hapi.humdata.org/docs#/Affected%20people/get_idps_api_v1_affected_people_idps_get).
+
+{{ read_yaml('data_usage_guides/endpoint_parameters/idps_parameters.yaml') }}
+
+### Transformations applied
+
+None
+
+### Usage notes
+
+* To obtain the number of IDPs per year in a given location,
+  only the most recent reporting round per operation should be aggregated
+
 ## Refugees & Persons of Concern <a id=”refugees”></a>
 
-This dataset, compiled by [UNHCR](https://www.unhcr.org/), offers annual
-age- and gender-disaggregated statistics on refugees and people of concern,
-categorised by their country of origin and country of asylum. The data are
-sourced primarily from governments hosting these populations, UNHCR's own
-registration data, and occasionally data published by non-governmental
-organizations.
+This sub-category is populated using data
+compiled by [UNHCR](https://www.unhcr.org/), which offers annual
+age- and gender-disaggregated global statistics on forcibly displaced and
+stateless persons, categorised by their country of origin and country
+of asylum. The data are sourced primarily from governments hosting
+these populations, UNHCR's own registration data, and occasionally
+data published by non-governmental organizations.
 
 ### Summary
 
-{{ read_yaml('data_usage_guides/subcategory_details/refugees_details.yaml') }}
+{{ read_yaml('data_usage_guides/subcategory_details/refugees_and_returnees_details.yaml') }}
```

Comment:
My understanding is that the refugees endpoint had some returnees data but now we have more. The refugees and returnees data are closely related.

Reply:
Prior to this change we were taking all the data from UNHCR, which contains refugees and POC, returnees, and IDPS, and putting it all under the "Refugees and POCs" endpoint. 

Now that we have DTM to populate the new IDP endpoint, we also created a "returnees" endpoint (to match the data grids), and the UNHCR data populates both the refugee and returnees.